### PR TITLE
Fix rng in binomial distribution

### PR DESCRIPTION
**Pull request by konimarti**
* rng function in Binomial distribution didn't work (variables were not initialized, etc.). I rewrote the rng algorithm and added tests to verify the random number generator.
* Tests have also been written for the rng function in the Poisson distribution
* Minor change in README to include multivariate normal in the Feature list

### CODE REVIEW

File: lib/rubystats/binomial_distribution.rb [hunk 1]
```diff
@@ -73,76 +73,37 @@ def get_cdf(_x)
       return sum
     end
 
-    # Inverse of the cumulative binomial distribution function 
-    # (equivalent to R qbinom function).
+    # Inverse of the cumulative binomial distribution function     
     # returns the value X for which P(x < _x).
-    def get_icdf(prob)
+    def get_icdf(prob)
```
Comment (phillbaker) [inline lib/rubystats/binomial_distribution.rb]: Please remove trailing whitespace

File: lib/rubystats/binomial_distribution.rb [hunk 2]
```diff
@@ -73,76 +73,37 @@ def get_cdf(_x)
...
+    def get_icdf(prob)   
       check_range(prob)
-      (find_root(prob, @n/2, 0.0, @n)).floor
+      sum = 0.0
+      k = 0
+      until prob <= sum 
+        sum += get_pdf(k)
+        k += 1
+      end 
+      return k - 1
```
Comment (phillbaker) [inline lib/rubystats/binomial_distribution.rb]: I don't think this `return` is necessary - the last statement is returned implicitly.

File: lib/rubystats/probability_distribution.rb [hunk 3]
```diff
@@ -129,8 +129,8 @@ def check_range(x, lo=0.0, hi=1.0)
     def get_factorial(n)
       if n <= 1
         return 1
-      else 
-        return n * get_factorial(n-1)
+      else
```
Comment (phillbaker) [inline lib/rubystats/probability_distribution.rb]: Please remove trailing white space

**Review by phillbaker — APPROVED** [review]
Thanks, just a few small style comments

**Comment by konimarti** [comment]
Travis checks failed because of a bug in earlier bundler versions. Travis file has been updated accordinly and all checks pass now.

**Comment by phillbaker** [comment]
Thanks!

**Pull request merged**